### ISSUE TITLE
Accessibility: Narrator doesn't announce collapsed/expanded input states on Chrome/Firefox

### ISSUE THREAD

**Issue by sajsarka** [OfficeDev/office-ui-fabric-react#11440]
### Environment Information

- **Package version(s)**: Visio Data Visualizer Excel Addin
- **Browser and OS versions**: Windows 10 : Chrome - 78.0.3904.108 , Firefox - 70.0.1

### Describe the issue:

Narrator is not announcing collapsed/expanded state on Chrome and Firefox.

#### Actual behavior:

Narrator is not announcing collapsed/expanded state on Chrome and Firefox.

#### Expected behavior:

Screen reader should announced the state of the combobox and dropdown after getting expanded and collapsed.
Please use the below codepen link to reproduce the issue.
https://codepen.io/sajsarka/pen/gObpLRg

**Comment by sajsarka** [comment]
@xugao : Reopen option was not there hence created new issue.

References : https://github.com/OfficeDev/office-ui-fabric-react/issues/11381 
https://github.com/OfficeDev/office-ui-fabric-react/issues/11367

**Comment by micahgodbolt** [comment]
Does this work in Edge? If it works in Edge but not Chrome, we would consider that an external issue with Chrome.

**Comment by sajsarka** [comment]
@micahgodbolt Yes it works on Edge.

**Comment by micahgodbolt** [comment]
We don't spend much time debugging narrator in other browsers, and wouldn't be able to provide much insight. I believe the best bet is to push back on those browser to better support the standard.

**Comment by sajsarka** [comment]
thanks @micahgodbolt for clearing that out. Any document which lists down different browser's accessibility behavior or support behavior could be very helpful. If any such document is there could you please share that.

**Comment by micahgodbolt** [comment]
if you see such a document i'd love to see it too :)
